apps/sms/views/conversation/js/conversation.js

```diff
@@ -692,11 +692,14 @@ var ConversationView = {
     // Add recipients and fill+focus the Compose area.
     var recipientPromise;
     if (activity.contact && number) {
-      recipientPromise = Utils.getContactDisplayInfo(findByAddress, number)
-        .then(function onData(data) {
-          data.source = 'contacts';
-          return data;
-        });
+      findByAddress(phoneNumber).then((contacts) => {
```

Comment:
you also need to assign the result of this line to `recipientPromise` just like the previous code did